nucypher/cli/commands/ursula.py

```diff
@@ -15,24 +15,33 @@
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 
 """
-import os
+
 
 import click
+import os
 from constant_sorrow.constants import NO_BLOCKCHAIN_CONNECTION
 
 from nucypher.blockchain.economics import EconomicsFactory
 from nucypher.blockchain.eth.utils import datetime_at_period
-from nucypher.characters.banners import URSULA_BANNER
-from nucypher.cli import actions, painting
-from nucypher.cli.actions import (
-    get_nucypher_password,
-    select_client_account,
-    get_client_password,
+from nucypher.cli.actions.auth import get_client_password, get_nucypher_password
+from nucypher.cli.actions.config import (
+    destroy_configuration,
     get_or_update_configuration,
-    select_config_file,
-    select_network)
+    get_provider_process,
+    handle_missing_configuration_file
+)
+from nucypher.cli.actions.network import determine_external_ip_address
+from nucypher.cli.actions.select import select_client_account, select_config_file, select_network
+from nucypher.cli.utils import make_cli_character, setup_emitter
```

Comment:
☝️ @KPrasch

Reply:
This kind of question is the essence of this PR.   "Utils" is always a junk-drawer-of-code. :joy:

Reply:
The main reason this function finds itself here Is because It does not quite qualify as an action IMO.  It's too complex and isn't focused around a descrete CLI "action" like *select, confirm, prompt, etc.* rather a *utility for making a character*.